examples/deploy_service/main.tf

```diff
@@ -19,12 +19,12 @@ locals {
 }
 
 provider "google" {
-  version = "~> 2.7.0"
+  version = "~> 2.9.0"
   region  = "${var.region}"
 }
 
 provider "google-beta" {
-  version = "~> 2.7.0"
+  version = "~> 2.9.0"
```

Comment:
2.9.1 is out ... any reason we do not use that??

Reply:
We don't require 2.9.1, so there's no reason to require it. This matches 2.9.X.